src/components/ParameterComponents/ParameterLabel.tsx

```diff
@@ -21,7 +21,7 @@ export default function ParameterLabel(props: {
   tooltip?: string;
 }): React.ReactElement {
   const tooltip: React.ReactElement[] = [];
-  if (props.tooltip) {
+  if (props.tooltip !== null) {
```

Comment:
I believe.that the remove of the `!=null` is to resolve ESLint error, does it throw error again?

Reply:
yeah seems that changing it to `!==null` fixes the issue.